.github/workflows/build-ale-py.yml

```diff
@@ -34,17 +34,19 @@ jobs:
     - name: Build wheels
       uses: pypa/cibuildwheel@v2.0.0
       env:
-        CIBW_MANYLINUX_X86_64_IMAGE: "ghcr.io/mgbellemare/manylinux2014-vcpkg"
-        CIBW_SKIP: "*i686* *win32*"
+        CIBW_MANYLINUX_X86_64_IMAGE: "ghcr.io/jessefarebro/manylinux2014-vcpkg"
+        CIBW_SKIP: "*i686* *win32* pp*"
+        CIBW_ARCHS_MACOS: "x86_64"
         CIBW_TEST_REQUIRES: "pytest"
         CIBW_TEST_COMMAND: "pytest {project}"
-        CIBW_ENVIRONMENT: "ALE_BUILD_VERSION=${{ env.ALE_BUILD_VERSION }} VCPKG_DEFAULT_TRIPLET=${{ matrix.config.triplet }}"
-        CIBW_BEFORE_ALL: "vcpkg install ${{ join(matrix.config.deps, ' ') }} --triplet ${{ matrix.config.triplet }}"
+        CIBW_ENVIRONMENT: "ALE_BUILD_VERSION=${{ env.ALE_BUILD_VERSION }} MACOSX_DEPLOYMENT_TARGET='10.15' VCPKG_DEFAULT_TRIPLET=${{ matrix.config.triplet }}"
```

Comment:
We have to set the min deployment target on macOS to 10.15 in order to get proper C++17 filesystem support.